tests/RobotFramework/tests/tedge_connect/tedge_connect_offline.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Setup
+Suite Teardown      Get Logs
+
+Test Tags           theme:mqtt    theme:c8y    adapter:docker
+
+
+*** Test Cases ***
+Check offline bootstrapping
+    ThinEdgeIO.Execute Command    sudo tedge disconnect c8y
+    ThinEdgeIO.Service Should Be Stopped    tedge-mapper-c8y
+
+    ThinEdgeIO.Disconnect From Network
```

Comment:
Where is defined this test command?

Reply:
It is part of the DeviceLibrary.py

Reply:
Under the hood the "disconnect" command is provided by the test adapter. For the container adapter, it uses docker api to disconnect the container from the docker network, https://docker-py.readthedocs.io/en/stable/networks.html#docker.models.networks.Network.disconnect

The container adapter code is reference [here](https://github.com/reubenmiller/device-test-core/blob/29af60f38e04427dc1cc5816229f375619de7f30/device_test_core/docker/device.py#L223)

Reply:
Thanks - This is clearer